packages/itwinui-react/src/core/Tooltip/Tooltip.tsx

```diff
@@ -196,8 +196,12 @@ const useTooltip = (options: TooltipOptions = {}) => {
       ...interactions.getReferenceProps(),
     }).forEach(([key, value]) => {
       if (typeof value === 'function') {
-        reference.addEventListener(domEventName(key), value);
-        cleanupValues[key] = value;
+        // manually add `.nativeEvent` (react concept) because floating-ui needs it
+        const patchedHandler = (event: Event) => {
```

Comment:
I saw that you mentioned in the PR description that without this change, some error was thrown. Just curious, under what circumstances was that error thrown? Was it thrown from some of our stories, examples from the docs site, etc.?

Reply:
it's thrown by the component when `reference` prop is used and you try to focus the trigger element.

i noticed it when one of the tests started failing and confirmed it in the playground. 

here's the source code in floating-ui which uses this property:
https://github.com/floating-ui/floating-ui/blob/cff301f4cb714ac0e699de5948460f1b8c571110/packages/react/src/hooks/useFocus.ts#L121